js/jquery.flot.tooltip.source.js

```diff
@@ -165,7 +165,11 @@
                         self.tooltips[STICKY_TOOLTIP].item = item;
                         delete self.tooltips[HOVER_TOOLTIP];
 
-                        self.plot.highlight(item.seriesIndex, item.dataIndex);
+                        // Need this delay to allow hideTooltip call above to unhighlight the series
+                        // This allows the dot marking the sticky tooltip datapoint to render
+                        window.setTimeout(() => {
+                            self.plot.highlight(item.seriesIndex, item.dataIndex);
+                        }, 0);
```

Comment:
Is it possible to maintain the dot independently for each chart? Right now if I stick a tooltip on chart #2 the dot on chart #1 disappears.

Reply:
I don't know how to do this currently or really how the dot is managed